public/index.php

```diff
@@ -3,12 +3,9 @@
 session_start();
 
 $config = require_once('../config.php');
-require_once '../MasterController.php';
+require_once '../vendor/autoload.php';
 
-require_once '../Comment.php';
-require_once '../User.php';
-require_once '../Story.php';
-require_once '../Index.php';
+require ('../diconfig.php');
 
-$framework = new MasterController($config);
-echo $framework->execute();
\ No newline at end of file
+$framework = new Masterclass\MasterController($di, $config);
```

Comment:
I did that originally, but then changed it to be injected into `RouteMap`.  This was so that I could use the DI config to build the routes.  Otherwise, I had built them in a separate file `routes.php` and included in the `index.php`.  

I could still do that... maybe it would be better.